projects/CalibrationStatusClassification/configs/t4dataset/resnet18_5ch_1xb8-25e_j6gen2.py

```diff
@@ -0,0 +1,165 @@
+    "../../../../autoware_ml/configs/calibration_classification/dataset/t4dataset/gen2_base.py",
+]
+
+custom_imports = dict(
+    imports=["autoware_ml.calibration_classification.datasets.transforms.calibration_classification_transform"],
+    allow_failed_imports=False,
+)
+
+data_root = "/workspace/data/t4dataset"
+info_directory_path = "/workspace/data/t4dataset/calibration_info/"
+train_projection_vis_dir = None
+val_projection_vis_dir = None
+val_results_vis_dir = None
+test_projection_vis_dir = "./test_projection_vis_t4dataset/"
+test_results_vis_dir = "./test_results_vis_t4dataset/"
+batch_size = 8
+num_workers = 8
+max_epochs = 25
+
+data_preprocessor = dict()
+
+model = dict(
+    type="mmpretrain.ImageClassifier",
+    backbone=dict(
+        type="ResNet",
+        depth=18,
+        in_channels=5,
+        num_stages=4,
+        out_indices=(3,),
+        frozen_stages=-1,
+        style="pytorch",
+    ),
+    neck=dict(type="GlobalAveragePooling"),
+    head=dict(
+        type="LinearClsHead",
+        num_classes=2,
+        in_channels=512,
+        loss=dict(type="CrossEntropyLoss", loss_weight=1.0),
+    ),
+)
+
+param_scheduler = dict(type="MultiStepLR", by_epoch=True, milestones=[5, 15, 20], gamma=0.1)
+
+train_cfg = dict(by_epoch=True, max_epochs=max_epochs, val_interval=1)
+
+train_pipeline = [
+    dict(
+        type="CalibrationClassificationTransform",
+        mode="train",
+        undistort=True,
+        enable_augmentation=False,
+        data_root=data_root,
+        projection_vis_dir=train_projection_vis_dir,
+        results_vis_dir=None,
+    ),
+    dict(
+        type="PackInputs",
+        input_key="fused_img",
+    ),
+]
+
+train_dataloader = dict(
+    batch_size=batch_size,
+    num_workers=num_workers,
+    persistent_workers=True,
+    shuffle=True,
+    dataset=dict(
+        type="T4CalibrationClassificationDataset",
+        ann_file=info_directory_path + _base_.info_train_file_name,
+        pipeline=train_pipeline,
+        data_root=data_root,
+    ),
+)
+
+val_cfg = dict()
+
+val_pipeline = [
+    dict(
+        type="CalibrationClassificationTransform",
+        mode="val",
+        undistort=True,
+        enable_augmentation=False,
+        data_root=data_root,
+        projection_vis_dir=val_projection_vis_dir,
+        results_vis_dir=val_results_vis_dir,
+    ),
+    dict(
+        type="PackInputs",
+        input_key="fused_img",
+        # meta_keys=["img_path", "fused_img", "image", "sample_idx", "frame_id", "frame_idx"],  # Visualization
+    ),
+]
+
+val_dataloader = dict(
+    batch_size=batch_size,
+    num_workers=num_workers,
+    persistent_workers=False,
+    shuffle=False,
+    dataset=dict(
+        type="T4CalibrationClassificationDataset",
+        ann_file=info_directory_path + _base_.info_val_file_name,
+        pipeline=val_pipeline,
+        data_root=data_root,
+    ),
+)
+
+val_evaluator = dict(topk=(1,), type="mmpretrain.evaluation.Accuracy")
+
```

Comment:
Did we miss `test_cfg()` definition?

```suggestion

test_cfg = dict()

```